src/zenml/step_operators/entrypoint.py

```diff
@@ -54,14 +53,14 @@ def create_executor_class(
     materializers = step_instance.get_materializers(ensure_complete=True)
 
     input_spec = {}
-    for input_name, class_path in input_artifact_type_mapping.items():
-        artifact_class = source_utils.load_source_path_class(class_path)
-        if not issubclass(artifact_class, BaseArtifact):
-            raise RuntimeError(
-                f"Class `{artifact_class}` specified as artifact class for "
-                f"input '{input_name}' is not a ZenML BaseArtifact subclass."
-            )
-        input_spec[input_name] = artifact_class
+    # for input_name, class_path in input_artifact_type_mapping.items():
```

Comment:
Should we remove these?

Reply:
Absolutely, I removed the comment and added an explanation why we can use just BaseArtifacts here